flytekit/models/interface.py

```diff
@@ -158,13 +158,19 @@ def from_flyte_idl(cls, proto: _interface_pb2.TypedInterface) -> "TypedInterface
             outputs={k: Variable.from_flyte_idl(v) for k, v in proto.outputs.variables.items()},
         )
 
-    def transform_interface_to_list(self) -> "TypedInterface":
+    def transform_interface_to_list(self, bound_inputs: typing.Set[str]) -> "TypedInterface":
         """
         Takes a single task interface and interpolates it to an array interface - to allow performing distributed
         python map like functions
+        :param bound_inputs: fixed inputs that should not be updated to a list and will be maintained as is
```

Comment:
What do you think about calling this "ignored_inputs":

```suggestion
        :param ignored_inputs: fixed inputs that should not be updated to a list and will be maintained as is
```

For this specific function, all it's doing is ignoring them. It does not have enough context to know what "bound" means.

Reply:
It's not ignoring the inputs, it's not changing it to a list<original_entity_type> instead keeping it as original_entity_type.

I actually have a follow up PR to fix a bug w/ fixed_inputs for LPs to add an excluded_inputs